esp/esp/users/templatetags/userlist.py

```diff
@@ -0,0 +1,11 @@
+from django.template import Library
+from django.utils.safestring import mark_safe
+
+register = Library()
+
+def userlist(var, delimiter):
+	atag = lambda x:'<a href="{0}">{1}</a>'.format(x.get_absolute_url(),unicode(x))
+	userlist_html = '{0} '.format(delimiter).join([atag(i) for i in var])
+	return mark_safe(userlist_html)
```

Comment:
Do we know that users can't set their name to contain HTML tags?  We should probably make sure it gets escaped on the way into this tag so that we can safely mark it as safe on the way out.